src/iface/ethernet.rs

```diff
@@ -353,6 +353,22 @@ impl<'b, 'c, 'e, DeviceT> Interface<'b, 'c, 'e, DeviceT>
         InterfaceInner::check_ethernet_addr(&self.inner.ethernet_addr);
     }
 
+    /// Get a reference to the inner device.
+    pub fn phy(&self) -> &DeviceT {
```

Comment:
Nit: I think this should be called `device`, since you aren't necessarily retrieving a phy.

Reply:
I didn't know on which part of the trait `phy::Device` to base the naming here, but that is no problem to change.

Reply:
Yeah, understandable, that entire part of hierarchy is a bit weird.